docs/output_cohorts_format.md

```diff
@@ -0,0 +1,167 @@
+Dump Cohort Output Format 
+========================
+
+This page explains the format of the file outputed when running mishegos.c. In order to improve the performance of mishegos the format of the cohort output changed the output format from JSON to dumpig the cohort structs directly. 
+
+Dumping the cohorts struct directly rewuires less serialization work and fewre allocations. In a seprate program called ___  can convert back to the JSON format. Inorder to be able to convert the file back into a JSON must know the format with the length and version prefix the structs. 
+
+##Format of the File 
+
+The file is organized in the following way: 
+
+* `nworkers` numbers of workers that will iterate through
+* `input` ? what does this do 
+* `input` other stuff in the input struct
+* For each worker there is: 
+	* `status` 0 or 1 
+	* `status_name` if status is 0 than unsuccessful and if status is 1 than successful
+	* `ndecoded` number of bytes decoded for the instruction
+	* `workerno` which number of worker on
+	* `worker_so` the path to the worker
+	* `len` the string length of the decoded instruction 
+	* `result` the output
+
+
+## Example of Output File - Usage 
+
+### Building
+
+As discussed in the mishegos.md file. `mishegos` is most easily build within Docker: 
+that 
+```bash
+docker build -t mishegos .
+```
+Alternatively, you can try building directly. See the README.md,
+
+###Running
+
+Run the fuzzer for a bit by: 
+
+```bash 
+./src/mishegos/mishegos ./workers.spec
+```
+
+`mishegos` checks for three environment variables:
+
+* `V=1` enables verbose output on `stderr`
+* `D=1` enables the "dummy" mutation mode for debugging purposes
+* `M=1` enables the "manual" mutation mode (i.e., read from `stdin`)
+
+
+Run the example given below
+
+```bash
+ M=1 ./src/mishegos/mishegos ./workers.spec <<< "36f03e4efd" > /tmp/mishego
+```
+
+###Example Output
+When opening up the `./tmp/mishegos` file in a text editor should see the following output: 
```

Comment:
I think we can drop this guidance: the output now contains non-printable characters, so opening it in a text editor probably won't help most people.